### ISSUE TITLE
Docs for conda based install missing IMSIM_HOME environment variable

### ISSUE THREAD

**Issue by esheldon** [LSSTDESC/imSim#440]
https://github.com/LSSTDESC/imSim/blob/345a7cc3307ce44b0e560efc0a68344523c18dbe/doc/install.rst?plain=1#L183

I believe there is a line missing  here, something like 

```bash
conda env config vars set IMSIM_HOME="directory containing the imSim github repo as a sub-directory"
```

**Comment by cwwalter** [comment]
Reminder to also look at the updating the skyCatalog instructions (or change the default sky catalog download branch) as it is currently grabbing main, which has issues.

**Comment by cwwalter** [comment]
This was fixed with #476.
